### PR TITLE
Update glossary file

### PR DESCRIPTION
**Pull request by imanalhasani**
translate the term "algorithmic bias" in line 636 to Arabic.

Fill in each of the sections (using NA for those that are not applicable).

## Author: 

Iman Al Hasani

## Language: 

Arabic

## Terms defined:

- algorithmic bias
- 
- 
-

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -642,7 +642,12 @@
     def: "አንድ ስልተ-ቀመር አንድን የተወሰነን ነገር ለማከናወን ሊከተሏቸው የሚገቡ እርምጃዎች ፣ ጥሰቶች ወይም ህጎች ስብስብነው
       \ አልጎሪዝም  በኮምፒተር ሳይንስ ውስጥ  የሂሳብ ችግሮችን ለመፍታት የተቀመጡ የኮምፒተር ፕሮግራም መመሪያዎች ስብስብ ነው
       \n"
-
+  ar:
+  term: " الخوارزمية المتحيزة"
```
Comment (BatoolMM) [inline glossary.yml:646]:
```suggestion
    term: " الخوارزمية المتحيزة"
```

**Review by BatoolMM — CHANGES_REQUESTED** [review]

**Comment by BatoolMM** [comment]
Hi Iman, 

Can you fill in the author, language and term in the description above, please? I also made a minor change for the space if you can approve it?

**Comment by imanalhasani** [comment]
Hi Batool 
I added the requested info.

**Review by imanalhasani — COMMENTED** [review]
I approved the changes done by Batool.

**Review by imanalhasani — COMMENTED** [review]
The block mapping is filled.

File: glossary.yml (hunk 1 above)
Comment (imanalhasani) [inline glossary.yml]:
```suggestion
  term: "الخوارزمية المتحيزة"
```

File: glossary.yml [hunk 2]
```diff
@@ -642,7 +642,12 @@
...
+  def: >
```
Comment (imanalhasani) [inline glossary.yml:647]:
```suggestion
   def: >
```

File: glossary.yml [hunk 3]
```diff
@@ -642,7 +642,12 @@
...
+  النتائج المتكررة التي تظهر التحيز أو المعالجة المنحرفة ، الموجودة في [الخوارزمية] (# الخوارزمية).
```
Comment (imanalhasani) [inline glossary.yml:648]:
```suggestion
  النتائج المتكررة التي تظهر التحيز أو المعالجة المنحرفة ، الموجودة في [الخوارزمية] (#الخوارزمية).
```

**Comment by BatoolMM** [comment]
@imanalhasani 
الملف تغير وبعض المصطلحات تغير مكانها .. تحتاجي ترجعي بالزمن لل commit السابقة .. خلينا نشتغل مع بعض عليه في جلسة يوم الجمعة

**Comment by imanalhasani** [comment]
Hello Batool

I added the requested information but I can not tick on the approval option
when I review the changes. Sorry you I am new to Github.

Thanks

**Comment by BatoolMM** [comment]
Hi @imanalhasani , no worries at all.

This PR is closed since August, did you submit another PR recently?

**Pull request closed without merging**